.github/workflows/create-release.yml

```diff
@@ -57,42 +61,29 @@
             fetch-depth: 0
 
         - name: Install uv
-          uses: astral-sh/setup-uv@v5.1.0
+          uses: astral-sh/setup-uv@v5
           with:
+            version: "0.5.28"  # install a specific version of uv
             enable-cache: true
 
         - name: Install the project
-          run: uv sync --dev
+          run: uv sync
 
         - name: Install graphviz
           run: sudo apt-get install -y graphviz graphviz-dev
 
         - name: Install erdantic
-          run: uv add "erdantic<1"
-
-        - name: Generate schema diagram files
-          run: |
-            uv run python -c 'import erdantic as erd
-            from psdm.steadystate_case.case import Case as SteadystateCase
-            from psdm.topology.topology import Topology
-            from psdm.topology_case.case import Case as TopologyCase
-            from pathlib import Path
-            f_top = Path(f"./schema/${{ steps.cz.outputs.version }}/topology.png")
-            f_top.parent.mkdir(exist_ok=True, parents=True)
-            erd.create(Topology).draw(f_top)
-            f_topc = Path(f"./schema/${{ steps.cz.outputs.version }}/topology_case.png")
-            f_topc.parent.mkdir(exist_ok=True, parents=True)
-            erd.create(TopologyCase).draw(f_topc)
-            f_ssc = Path(f"./schema/${{ steps.cz.outputs.version }}/steady_state_case.png")
-            f_ssc.parent.mkdir(exist_ok=True, parents=True)
-            erd.create(SteadystateCase).draw(f_ssc)'
-
-        - name: Copy diagrams to docs
-          run: yes | cp -rf ./schema/${{ steps.cz.outputs.version }}/*.png ./docs/
+          run: uv add "erdantic>=1.0.5"
```

Comment:
you don't run it in a separate workflow. you need to upload this as an artifact in the other workflow and access it here to really separate it.

Reply:
create-class-diagram resp. erdantic-plugin is not triggered anylonger.
Instead only the latest class diagrams (stored in ./docs) are copied to the newly created schema version directory.

This is sufficient as the create-class-diagram action is run on every PR review event.